modules/vectra.py

```diff
@@ -1895,22 +1791,15 @@ def get_account_by_id(self, account_id=None, **kwargs):
             tags, note, note_modified_by, note_modified_timestamp, privilege_level, privilege_category, 
             last_detection_timestamp, detection_set, probable_home
         """
-        if not account_id:
-            raise ValueError('Account id required')
-
-        return requests.get('{url}/accounts/{id}'.format(url=self.url, id=account_id), headers=self.headers,
-                                params=self._generate_account_params(kwargs), verify=self.verify)
+        raise DeprecationWarning('Thismethod no longer functions correctly, please use API > v2.2')
```

Comment:
Missing space in the string.  I would simplify it to 'This method is deprecated.  Use API Version > 2.2'